### ISSUE TITLE
Could use a nicer way of writing empty properties for DUs

### ISSUE THREAD

**Issue by haf** [xyncro/chiron#26]
Like:

``` fsharp
type T =
 | A
 | B
 static member ToJson (t : T) =
  match t with
  | A -> Json.write "a" (None : string option)
  | B -> Json.write "b" (None : string option)
```

instead perhaps

`Json.writeNullProp "a"`

**Comment by kolektiv** [comment]
Yes, that could be handy. Will add.

**Comment by haf** [comment]
Ping?

**Comment by kolektiv** [comment]
Will try this and if it works nicely will include it in 5.0

**Comment by kolektiv** [comment]
https://github.com/kolektiv/chiron/commit/ad5bf47d25920ef5f1e469725e01b3d8a01b346e includes `Json.writeNone` for such cases. Although wondering if `writeNull`would be better...

**Comment by haf** [comment]
Yes, probably. Nice

**Comment by CumpsD** [comment]
Seems this is released and working, just tried it in 6.0.1

`Json.writeNone "bla"` results in:
```
"hash": {
   "bla": null
},
```

Probably can close
